lighthouse/classes/services/cherrytrack.py

```diff
@@ -0,0 +1,37 @@
+from lighthouse.helpers.cherrytrack import (
+    get_automation_system_run_info_from_cherrytrack,
+    get_samples_from_source_plate_barcode_from_cherrytrack,
+)
+import logging
+from http import HTTPStatus
+
+logger = logging.getLogger(__name__)
+
+
+class ServiceCherrytrackMixin(object):
```

Comment:
I would rename this to CherrytrackServiceMixin to be consistent with the events etc. The warehouse service would also be renamed. I think the convention should be `<name>ServiceMixin`.

Reply:
ok, I'll change it